src/acceleration/impl/ResidualSumPreconditioner.cpp

```diff
@@ -45,20 +45,31 @@ void ResidualSumPreconditioner::_update_(bool                   timestepComplete
       norms[k] = std::sqrt(norms[k]);
     }
     sum = std::sqrt(sum);
-    PRECICE_CHECK(not math::equals(sum, 0.0), "All residual sub-vectors in the residual-sum preconditioner are numerically zero. "
-                                              "Your simulation probably got unstable, e.g. produces NAN values.");
+    if (math::equals(sum, 0.0)) {
+      PRECICE_WARN("All residual sub-vectors in the residual-sum preconditioner are numerically zero ( sum = " << sum << "). This indicates that the data values exchanged between two succesive iterations did not change."
+                                                                                                                         " The simulation may be unstable, e.g. produces NAN values. Please check the data values exchanged"
+                                                                                                                         " between the solvers is not identical between iterations.");
```

Comment:
As this is now a warning and not an error, we need to tell the user what is happening (i.e. no re-scaling etc).
Btw, you can improve readability of the formater with an additional line break.